tests/travis/docker-run-mysql-or-mariadb.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Starting RDBMS…"
+
+if [[ "$IMAGE" == "mysql:8.0" ]]
+then
+  CMD_OPTIONS="--default-authentication-plugin=mysql_native_password"
+else
+  CMD_OPTIONS=""
+fi
+
+docker run \
+    --health-cmd='mysqladmin ping --silent' \
+    --detach \
+    --env MYSQL_ALLOW_EMPTY_PASSWORD=yes \
+    --env MYSQL_DATABASE=doctrine_tests \
+    --publish 33306:3306 \
+    --name rdbms \
+    "$IMAGE" $CMD_OPTIONS
+
+function healthStatus()
+{
+  docker inspect --format "{{json .State.Health.Status }}" rdbms
+}
+
+until [ "$(healthStatus)" == '"healthy"' ]
+do
+  if [ "$(healthStatus)" == '"unhealthy"' ]
```

Comment:
Given that there are only three possible statuses, could this be optimized and simplified using the following logic (didn't test it)?
```sh
while true; do
    $status = $(docker inspect --format "{{json .State.Health.Status }}" rdbms)

    if [ "$status" == '"healthy"' ]
    then
        break
    fi

    if [ "$status" == '"unhealthy"' ]
    then
        exit 1
    fi

    echo "Waiting for RDBMS to become ready (status = $status)…"
    sleep 1
done
```

Reply:
I avoided `status` because it seems to be reserved in some shells (zsh?): vim colors it differently than other names, and it is defined in my shell, although I have many plugins.

Reply:
Also used a switch case to avoid an infinite loop